pkg/operator/ceph/controller/predicate.go

```diff
@@ -135,11 +125,12 @@ func WatchControllerPredicate() predicate.Funcs {
 					}
 					return true
 				}
-				// Handling upgrades
-				isUpgrade := isUpgrade(objOld.GetLabels(), objNew.GetLabels())
-				if isUpgrade {
-					return true
-				}
+
+			case *cephv1.CephCluster:
```

Comment:
The file/object/nfs controllers will all be informed immediately now, right? Should we still wait for the mons to be updated first before updating the child daemons?

Reply:
Yes. It is difficult to wait for the monitors at this point.